app/services/db/monthly_sp_auth_count/new_unique_monthly_user_counts_by_partner.rb

```diff
@@ -82,6 +82,20 @@ def call(partner:, issuers:, start_date:, end_date:)
             end
           end.tap { |counts| counts.default = [] }
 
+          new_unique_profile_age_counts = new_unique_user_proofed_events.group_by do
+                                          |user_unique_id|
+            age = users_to_profile_age[user_unique_id]
+            if age.nil? || age < 0
+              :unknown
+            elsif age > 4
+              :older
+            else
+              age.to_i
+            end
+          end.tap do |counts|
+            counts.default = []
+          end
```

Comment:
this is the exact same giant block as before, seems like a great candidate for a helper

```suggestion
          new_unique_profile_age_counts = group_by_profile_age(
            events: new_unique_user_proofed_events,
            users_to_profile_age:
          )
```

Reply:
Do you mean just a function or a separate helper in like a separate file? Good item to discuss in pairing?

Reply:
just a method in the same file